maths/area.py

```diff
@@ -458,6 +458,49 @@ def area_rhombus(diagonal_1: float, diagonal_2: float) -> float:
     return 1 / 2 * diagonal_1 * diagonal_2
 
 
+def area_reg_polygon(sides: int, length: float) -> float:
+    """
+    Calculate the area of a regular polygon.
+    Wikipedia reference: https://en.wikipedia.org/wiki/Polygon#Regular_polygons
+    Formula: (n*s^2*cot(pi/n))/4
+    >>> area_reg_polygon(3, 10)
+    43.301270189221945
+    >>> area_reg_polygon(4, 10)
+    100.00000000000001
+    >>> area_reg_polygon(0, 0)
+    Traceback (most recent call last):
+        ...
+    ValueError: area_reg_polygon() only accepts integers greater than or equal to \
+three as number of sides
+    >>> area_reg_polygon(-1, -2)
+    Traceback (most recent call last):
+        ...
+    ValueError: area_reg_polygon() only accepts integers greater than or equal to \
+three as number of sides
+    >>> area_reg_polygon(5, -2)
+    Traceback (most recent call last):
+        ...
+    ValueError: area_reg_polygon() only accepts non-negative values as \
+length of a side
+    >>> area_reg_polygon(-1, 2)
+    Traceback (most recent call last):
+        ...
+    ValueError: area_reg_polygon() only accepts integers greater than or equal to \
+three as number of sides
+    """
+    if not isinstance(sides, int) or sides < 3:
+        raise ValueError(
+            "area_reg_polygon() only accepts integers greater than or \
+equal to three as number of sides"
+        )
+    elif length < 0:
+        raise ValueError(
+            "area_reg_polygon() only accepts non-negative values as \
+length of a side"
+        )
+    return (sides * length**2) / (4 * tan(pi / sides))

```

Comment:
```suggestion
    return (sides * pow(length, 2)) / (4 * tan(pi / sides))
```

Reply:
Hi @CaedenPH , Based on the format of preexisting functions, I doubt if I'm allowed to commit your suggestion.